media-libs/aribb24/aribb24-1.0.3.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools
+
+DESCRIPTION="Library for decoding ARIB STD-B24 subtitles"
+HOMEPAGE="https://github.com/nkoriyama/aribb24"
+SRC_URI="https://github.com/nkoriyama/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-3"
```

Comment:
AFAICS most of the source files specify LGPL-2.1+ in the heading. Could you verify this? If all sources use LGPL-2.1+, then that is the correct license

Reply:
The files that do have headers say LGPL-2.1+, but I'm reluctant to claim that as the license when the main COPYING file is LGPL-3 (and some files have no license header, so the COPYING file should apply as a whole).

Reply:
Are those files with no license header sources or otherwise installed?

Reply:
At least `aribb24.pc.in` has no header and ends up installed (as /usr/lib64/pkgconfig/aribb24.pc). It's minor though.

Reply:
Pkg-config files are generally non-copyrightable (they contain only build instructions, nothing really original).

Reply:
Is there a policy about this? I understand the logic that the lowest common denominator of the source files is the real license, but I figure there has to be a good reason to ignore the declared package license in favor of our own investigation. LGPL-3 is a superset of LGPL-2.1+ so it should be safer to say the package is LGPL-3.

Reply:
Yeah, that's good enough for now. I've filed a bug upstream to verify it.